ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentDetails.tsx

```diff
@@ -33,7 +33,8 @@ import { ReactComponent as BothPolicyRules } from 'images/network-graph/both-pol
 import { ReactComponent as EgressOnly } from 'images/network-graph/egress-only.svg';
 import { ReactComponent as IngressOnly } from 'images/network-graph/ingress-only.svg';
 import { ReactComponent as NoPolicyRules } from 'images/network-graph/no-policy-rules.svg';
-import { deploymentTabs } from '../utils/deploymentUtils';
```

Comment:
Bravo! This file has become an orphan.

We can discuss on call: pro and con to delete interaction to switch tabs (because of RBAC) now or later.